yarn.lock

```diff
@@ -841,6 +841,11 @@
   dependencies:
     any-observable "^0.3.0"
 
+"@sindresorhus/is@^0.14.0":
```

Comment:
Should we really be checking in the `yarn.lock` file? Do we have any consensus on handling this file?

Reply:
That's a fundamental point of a npm/yarn lock files to know when deps are changing. Thought realistically we should use specific versions in package.json.